README.md

```diff
@@ -67,11 +70,29 @@ The feature management library supports appsettings.json as a feature flag sourc
+            "RequirementType": "All",
+            "EnabledFor": [
+                {
+                    "Name": "TimeWindow",
+                    "Parameters": {
+                        "Start": "Wed, 01 May 2019 13:59:59 GMT",
+                        "End": "Mon, 01 July 2019 00:00:00 GMT"
+                    }
+                },
+                {
+                    "Name": "Percentage",
+                    "Parameters": {
+                        "Value": "50"
+                    }
+                }
+            ]
+        }
     }
 }
 ```
 
-The `FeatureManagement` section of the json document is used by convention to load feature flag settings. In the section above, we see that we have provided three different features. Features define their feature filters using the `EnabledFor` property. In the feature filters for `FeatureT` we see `AlwaysOn`. This feature filter is built-in and if specified will always enable the feature. The `AlwaysOn` feature filter does not require any configuration so it only has the _Name_ property. `FeatureU` has no filters in its `EnabledFor` property and thus will never be enabled. Any functionality that relies on this feature being enabled will not be accessible as long as the feature filters remain empty. However, as soon as a feature filter is added that enables the feature it can begin working. `FeatureV` specifies a feature filter named `TimeWindow`. This is an example of a configurable feature filter. We can see in the example that the filter has a parameter's property. This is used to configure the filter. In this case, the start and end times for the feature to be active are configured.
+The `FeatureManagement` section of the json document is used by convention to load feature flag settings. In the section above, we see that we have provided four different features. Features define their feature filters using the `EnabledFor` property. In the feature filters for `FeatureT` we see `AlwaysOn`. This feature filter is built-in and if specified will always enable the feature. The `AlwaysOn` feature filter does not require any configuration so it only has the _Name_ property. `FeatureU` has no filters in its `EnabledFor` property and thus will never be enabled. Any functionality that relies on this feature being enabled will not be accessible as long as the feature filters remain empty. However, as soon as a feature filter is added that enables the feature it can begin working. `FeatureV` specifies a feature filter named `TimeWindow`. This is an example of a configurable feature filter. We can see in the example that the filter has a parameter's property. This is used to configure the filter. In this case, the start and end times for the feature to be active are configured. `FeatureW` specifies a requirement type of `All`, meaning all of it's filters must return true for the feature to be active. In this case, the feature is active between the specified time window and for 50% of users.

```

Comment:
Same comment here about "returning true" in FeatureW explanation - we can say "enabled" instead.
And same comment about`RequirementType` in quotes.